shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt

```diff
@@ -42,9 +42,9 @@ object ApiUtils {
         mapper.readValue(content, Notification::class.java)
 }
 
-fun String.parseTimeParameter(errorMsg: String): OffsetDateTime =
+fun String.parseTimeParameter(errorMsg: String): ZonedDateTime =
     try {
-        OffsetDateTime.parse(this)
+        ZonedDateTime.parse(this)
```

Comment:
This should be converted to UTC or no?

Reply:
No, since we are not managing yet the conversion of user input.